README.md

```diff
@@ -123,15 +154,24 @@ Response:
 }
 ```
 
+- **pid** is the process id of the zmachine instance that you sent it to. Will be
+ the same as the pid in the URL you posted to.
+- **data** is the response the game returned when it was saved. Don't send this
+ to the player, as they don't know we're saving and restoring repeatedly.
```

Comment:
I would say the second sentence should probably live in the tutorial below, since it may not be true depending on the client's choices in managing game instances.